script/runc-centos.spec

```diff
@@ -0,0 +1,55 @@
+##  Build docker runc RPM.
+##   1. Copy runc project tarball into ~/rpmbuild/BUILD/docker-runc.tar.gz
+##   2. run rpmbuild -ba script/runc.spec  -> will build source and binary package both.
+##
+##  Note:
+##   1. tarball must be a git project and will checkout the version while building rpms.
+##   2. this spec will build runc in dynamic link not static (if static is need, just uncomment build rules)
+##   3. need to update the TAG of runc project if release a rpm package.
+##
+%global _bindir /usr/local/bin
+%global _tag   v1.0.0-rc1
+
+Name: docker-runc
+Version: 1.0.0
+Release: 1%{?dist}
+Summary: runc is a CLI tool for spawning and running containers according to the OCF specification
+
+License: ASL 2.0
+Source: %{name}.tar.gz
+
+URL: https://www.opencontainers.org/
+Vendor: OCI
+Packager: OCI 
+
+BuildRequires: glibc
+#BuildRequires: glibc-static
+BuildRequires: make
+BuildRequires: libseccomp-devel 
+BuildRequires: libselinux-devel
+BuildRequires: git
+
+
+%description
+runc is a CLI tool for spawning and running containers according to the OCF specification
+
+%prep
+%setup -c -n runc
+
+%install
+git checkout -q %{_tag}
+make BUILDTAGS="seccomp selinux"
+#make BUILDTAGS="seccomp selinux" static
```

Comment:
Commented line is the way to build static, if someone need this, I think he/she just uncomment it will be OK.